.chloggen/961.yaml

```diff
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
I recognize that the specification allows this as a non-breaking change because it is an attribute value, but I am very concerned about the end user experience of this change.  This change will break alerts/slos/boards that expect query parameters to be present in the attribute named `url.full` or `url.query`.

Reply:
Especially painful for users who are treating the query parameters as non-sensitive data

Reply:
Yeah, in practice this can be a pretty substantial breaking change for end users, most of whom likely don't have sensitive information in these query params